### ISSUE TITLE
model performance

### ISSUE THREAD

**Issue by aurora5161** [microsoft/qlib#889]
hi author:
According to https://github.com/microsoft/qlib/blob/main/examples/benchmarks/README.md, if we consider Annualized Return as the main metric in alpha158 dataset, We found that tree model is better than time serial model. With alpha360 dataset, Transformer and TabNet's Annualized Return is unexpectedly negative. Can you give me some advice about performance evaluation method of quant. In News recommendation scene, we use AUC metric to evaluate model performance. what's the most metric of quant? thank you for your reply.

**Comment by you-n-g** [comment]
[First,](https://github.com/microsoft/qlib/pull/899)  We have very limited resources to implement and finetune the models. We tried our best effort to fairly compare these models.  But some models may have greater potential than what it looks like in the table below.  Your contribution is highly welcomed to explore their potential.

Personally, 
I think the tree model often beats neural networks on the tabular datasets with less spatial relationships, which Alpha158 belongs to. 

Alpah360 is more like raw price and volume data with strong spatial relationships in the time dimension. Neural networks designed for time-series data will be good choices for them.

Signal based evaluation and portfolio based metrics are the most frequently used in Quant research

**Comment by aurora5161** [comment]
> 
It helps me a lot and thank you very much!
